DeepSpeech.ipynb

```diff
@@ -1268,11 +1253,7 @@
    },
    "outputs": [],
    "source": [
-    "def run_inference(session, caption, data_set, results_params=None):\n",
-    "    if results_params is None:\n",
-    "        # Get the data_set specific graph end-points\n",
-    "        results_params = get_results_params(data_set)\n",
-    "        \n",
+    "def run_inference(session, caption, data_set, feed_dict, results_params):\n",
```

Comment:
Could you change the signature to something like
```python
run_inference(session, caption, total_batches, feed_dict, results_params)
```
as you only use `data_set.total_batches`. This will make the intention of the method clearer.